### DISCUSSION TITLE
Image is checkered

### DISCUSSION THREAD

**Discussion by alidanial7** [k-yle/rtsp-relay#67] [Q&A] [answered]
**Hi**
I'm start  using your package and I love it but i have a problem that i couldn't solve.
when i connect to camera for about 1 second every thing is good but after that image goes to be checkered but rtsp works fine in vlc.
is there any to fix this?
![123](https://user-images.githubusercontent.com/30351558/120095136-7c70dc80-c139-11eb-9d70-3d49a9ed1dcb.png)

**Comment by baur** [comment, marked as answer]
@alidanial7 

[Improving the video quality](https://github.com/k-yle/rtsp-relay#improving-the-video-quality)

It helped me

**Reply by alidanial7** [reply]
The image that i attached is with that configurations :(

**Reply by baur** [reply]
It may be related to client js player
